medical_ui/catalog/views.py

```
@@ -0,0 +1,72 @@
+
+import importlib.util
+from pathlib import Path
+from typing import Any
+
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+
+from .forms import ModelDiscoveryForm
+
+
+def _load_medical_script() -> Any:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_load_medical_script`

(ANN401)